dask/blockwise.py

```diff
@@ -243,6 +251,14 @@ def _out_numblocks(self):
 
         return out_d
 
+    def get_dependencies(self, all_hlg_keys):
+        _ = self._dict  # trigger materialization
+        return self._cached_dict["basic_layer"].get_dependencies(all_hlg_keys)
```

Comment:
Eventually I think that we're going to want to separate dependencies from graph generation.  This can safely be future work though.

Reply:
Agree and agree :)